scripts/pepper

```diff
@@ -20,22 +29,61 @@ logging.basicConfig(format='%(levelname)s %(asctime)s %(module)s: %(message)s')
 logger = logging.getLogger('pepper')
 logger.addHandler(NullHandler())
 
+
+class Pepper(object):
+    def __init__(self):
+        self.cli = PepperCli()
+        self.opts = salt.config.client_config(self.cli.options.master)
+
+    @property
+    def output(self):
+        if not hasattr(self, 'modules'):
+            self.modules = salt.loader.minion_mods(self.opts)
+        try:
+            oput = self.modules[self.cli.args[1]].__outputter__
```

Comment:
This is my biggest concern, if they pass more than one module on the commandline.

Reply:
Although salt-api can invoke multiple modules in a single req/resp, I don't recall that Pepper exposes that...

Reply:
I will double check, but yeah i did not see anything that would send multiple modules as one command